akka-docs/src/main/paradox/scala/cluster-team.md

```diff
@@ -1,15 +1,109 @@
+The grouping of nodes into teams is not limited to the physical boundaries of data centers,
+even though that is the primary use case. It could also be used as a logical grouping
+for other reasons, such as isolation of certain nodes to improve stability or splitting
+up a large cluster into smaller groups of nodes for better scalability. 
+
+## Membership
+
+Some @ref:[membership transitions](common/cluster.md#membership-lifecycle) are managed by 
+one node called the @ref:[leader](common/cluster.md#leader). There is one leader per team and
+it is responsible for these transitions for the members within the same team. Members of
+other teams are managed independently by other team leaders. These actions cannot be performed
+while there are any unreachability observations among the nodes in the team, but unreachability
+across different teams don't influence the progress of membership management within a team.
+Nodes can be added and removed also when there are network partitions between data centers,
+which is impossible if nodes are not grouped into teams.
+
+![cluster-team.png](../images/cluster-team.png)
+
+User actions like joining, leaving, and downing can be sent to any node in the cluster,
+not only to the nodes in the team of the node. Seed nodes are also global.
+
+The team membership is implemented by adding the team name prefixed with `"team-"` to the 
+@ref:[roles](cluster-usage.md#node-roles) of the member and thereby this information is known
+by all other members in the cluster.
+
+## Failure Detection
+
+@ref:[Failure detection](cluster-usage.md#failure-detector) is performed by sending heartbeat messages
+to detect if a node is unreachable. This is done more frequently and with more certainty among
+the nodes in the same team than across teams. The failure detection across different teams should
+be interpreted as an indication of problem with the network link between the data centers.
+
+Two different failure detectors can be configured for these two purposes:
+
+* `akka.cluster.failure-detector` for for failure detection within own team
+* `akka.cluster.inter-team-failure-detector` for for failure detection across different teams
+
+When @ref:[subscribing to cluster events](cluster-usage.md#cluster-subscriber) the `UnreachableMember` and
+`ReachableMember` events are for observations within the own team. The same team as where the
+subscription was registered.
+
+For cross team unreachability notifications you can subscribe to `UnreachableTeam` and `ReachbleTeam`
+events.
+
+## Cluster Singleton
+
+The [Cluster Singleton](cluster-singleton.md) is a singleton per team. If you start the 
+`ClusterSingletonManager` on all nodes and you have defined 3 different teams there will be
+3 active singleton instances in the cluster, one in each team. This is taken care of automatically, 
+but is important to be aware of.
+
+The reason why the singleton is per team and not global is that there is not enough consistency
```

Comment:
Maybe instead of "that there is not enough consistency in the membershipt management" -> "that membership information is not guaranteed to be consistent across teams when using ..."